desktop-sync/latest/support/index.md

```diff
@@ -2,13 +2,10 @@
 title: Supported platforms
 ---
 
-The following are the supported platforms for Desktop Sync 1.14:
+The following are the supported platforms for Desktop Sync 1.15:
 
 | Version | Notes |
 | ------- | ----- |
-| Content Services 7.3.x | |
```

Comment:
@anxumalo I am not sure why have we removed earlier ACS versions. Any reason?

Reply:
Just a suggestion - should I put 7.3 back in?

Reply:
I think we can put it back. With this change, we will have only one version of ACS i.e 7.4 supporting Desktop Sync. I am not sure whether it's worth keeping only one version or not. Maybe after 23.X we can remove versions older than 7.4. It's better to keep at least 2-3 supported versions at least.

Reply:
I've kept 7.3 & 7.2 - should i do the same for Sync Service  #963?

Reply:
I think so, we can keep it.